### PR TITLE
Adds some additional cleanables for cleanbots

### PR DESCRIPTION
**Pull request by Anonmare**
:cl:
refactor: Cleanbots are now more discriminating in what they consider trash
/:cl:

why: Just seemed like it would be nice.

### CODE REVIEW

File: code/modules/mob/living/simple_animal/bot/cleanbot.dm [hunk 1]
```diff
@@ -201,6 +202,11 @@
 
 	if(trash)
 		target_types += /obj/item/trash
+		target_types += /obj/item/cigbutt
+		target_types += /obj/item/ammo_casing/spent
+		target_types += /obj/item/light/tube/broken
+		target_types += /obj/item/light/bulb/broken
+		target_types += /obj/item/reagent_containers/food/snacks/deadmouse
```
Comment (coiax) [inline code/modules/mob/living/simple_animal/bot/cleanbot.dm]: Feels like these two lists could probably be unified.

**Comment by Tlaltecuhtli** [comment]
add limbs and organs too,  its probably the most common kind of trash in the station

**Comment by Vidiveni** [comment]
Banana peels.

File: code/modules/mob/living/simple_animal/bot/cleanbot.dm [hunk 2]
```diff
@@ -130,7 +130,8 @@
 		target = scan(/obj/effect/decal/remains)
 
 	if(!target && trash) //Then for trash.
-		target = scan(/obj/item/trash)
+		target = scan(/obj/item/trash, /obj/item/cigbutt, /obj/item/ammo_casing/spent, /obj/item/light/tube/broken,
```
Comment (ShizCalev) [inline code/modules/mob/living/simple_animal/bot/cleanbot.dm]: this doesn't do what you think it does.

**Review by ShizCalev — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/simple_animal/bot/cleanbot.dm (hunk 2 above)
Comment (ShizCalev) [reply code/modules/mob/living/simple_animal/bot/cleanbot.dm]: https://github.com/tgstation/tgstation/blob/a9ca93b60dbf7b9f0fc5cbe94c406a2126312fee/code/modules/mob/living/simple_animal/bot/bot.dm#L403

**Comment by drderp3635** [comment]
in the changelog it should be less discriminating, not more discriminating

**Comment by Anonmare** [comment]
Need more time to figure out how cleanbot code works

File: code/modules/mob/living/simple_animal/bot/cleanbot.dm (hunk 2 above)
Comment (tortellinitony) [reply code/modules/mob/living/simple_animal/bot/cleanbot.dm]: change to /obj/item I'd think

**Pull request closed without merging**